src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -124,7 +124,7 @@ def merge_hosts_by_hostname(ceph_hosts, orch_hosts):
 
     :param ceph_hosts: hosts returned from mgr
     :type ceph_hosts: list of dict
-    :param orch_hosts: hosts returned from ochestrator
+    :param orch_hosts: hosts returned from orchestrator
```

Comment:
I typically tag things like this w/ `brand`, but this isn't technically a brand. It is amusing, as #51226 is a separate PR at this point.